src/libmongoc/src/mongoc/mongoc-stream-tls-secure-channel.c

```diff
@@ -995,8 +995,9 @@ mongoc_stream_tls_secure_channel_new (mongoc_stream_t *base_stream,
       mongoc_secure_channel_setup_crl (secure_channel, opt);
    }
 
+   PCCERT_CONTEXT cert = NULL;
```

Comment:
I'd suggest moving this to the top of the function declaration for consistency. Otherwise, this looks good.